src/setlistSecondary.java

```diff
@@ -0,0 +1,104 @@
+package components.setlist;
+
+import java.util.Objects;
+
+/**
+ * Abstract secondary implementation of {@code Setlist}, layering over
+ * {@code SetlistKernel}. Author: Alex Desmarais
+ */
+public abstract class setlistSecondary implements setlist {
```

Comment:
Not a huge deal, but we usually user UpperCasing for classes/interfaces. 
```suggestion
public abstract class SetlistSecondary implements Setlist {
```